packages/hooks/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 #### Breaking Changes
 
+- Update `useConfig` return parameter from `Promise<Config>` to `{ data?: Config, error?: Error }` [#120](https://github.com/liteflow-labs/liteflow-js/pull/120)
+  - Make sure to update your codebase with `const { data } = useConfig();`
+
```

Comment:
Missing removal of `config` props in `LiteflowContext.Provider`

Reply:
done